src/main/java/appeng/core/sync/packets/PacketNEIBookmark.java

```diff
@@ -67,42 +63,13 @@ public void serverPacketData(final INetworkInfo manager, final AppEngPacket pack
         final Container con = pmp.openContainer;
 
         if (con instanceof ContainerMEMonitorable monitorable) {
-            final IGridNode node = monitorable.getNetworkNode();
-            if (node != null) {
-                final IGrid grid = node.getGrid();
-                if (grid == null) {
-                    return;
-                }
-
-                final IStorageGrid inv = grid.getCache(IStorageGrid.class);
-                final IMEMonitor<IAEItemStack> storage = inv.getItemInventory();
-                final IEnergyGrid energy = grid.getCache(IEnergyGrid.class);
+            final IMEMonitor<IAEItemStack> monitor = monitorable.getMonitor();
+            if (monitor != null) {
+                final IEnergySource energy = monitorable.getPowerSource();
                 final BaseActionSource actionSource = monitorable.getActionSource();
 
                 final AEItemStack request = AEItemStack.create(bookmarkItem);
-                final IAEItemStack out = Platform.poweredExtraction(energy, storage, request, actionSource);
-                if (out != null) {
-                    final InventoryAdaptor adp = InventoryAdaptor.getAdaptor(player, ForgeDirection.UNKNOWN);
-                    ItemStack outItem = out.getItemStack();
-                    adp.addItems(outItem);
-                }
-            }
-        }
-        if (con instanceof ContainerWirelessTerm wirelessTerm) {
```

Comment:
I vaguely recall there being a problem where the wireless term had a different monitor than the one it overrode. Does using the get monitor method manage to get around that problem?

Reply:
@firenoo is this fixed or why it show outdated when i view it?

Reply:
@firenoo yup the getter method works for the wireless terminal